src/engine/Settings.cs

```diff
@@ -830,6 +830,15 @@ public void ApplyLanguageSettings()
         GD.Print("Set C# locale to: ", cultureInfo, " Godot locale is: ", TranslationServer.GetLocale());
     }
 
+    /// <summary>
+    ///   Currently only used by one dialog, called to update the specified setting without using the dedicated menu
+    /// </summary>
+    public void UpdateSettingExternally(SettingValue<bool> setting, bool value)
```

Comment:
I'm a bit unsure why this method was required to exist. Does it not work just by setting the value directly on that one option?